movie-browser/src/components/Featured.js

```diff
@@ -1,18 +1,13 @@
 import React, { Component } from 'react';
+import Popup from 'reactjs-popup'
 
 class Featured extends Component {
-    constructor(props) {
-        super(props);
+      state = { movies:[] }
 
-        this.state = {
-            movies: []
-        };
-    }
-  
     componentDidMount() {
       fetch('https://ghibliapi.herokuapp.com/films')
-        .then(response => response.json())
-        .then(movies => this.setState({ movies }));
+      .then(response => response.json())
+      .then(data => this.setState({ movies: data }));
```

Comment:
you should use your api's `fetchMovies` here

Reply:
I will refactor into using the API.js component. I wanted the app to work first.

Reply:
alright